src/services/device.service.ts

```diff
@@ -0,0 +1,23 @@
+import { db } from './database';
+
+export interface Device {
+  deviceId: number;
+  userId: number;
+  deviceToken: number;
```

Comment:
I think this need to be:

```suggestion
  deviceToken: string;
```

I looked up what these tokens will look like, in case you're interested: https://stackoverflow.com/q/39959417